### PR TITLE
[SUCE-433] all logos updated

### PR DESCRIPTION
**Pull request by SamaneriVimala**
Some minor changes to css, added B&W logo for sidebar and uploaded the whole package for all platforms of the favicon.

### CODE REVIEW

File: client/elements/sc-drawer-layout.html [hunk 1]
```diff
@@ -81,9 +82,9 @@
         height: 100%;
       }
 
-      .nav-home-title-text {
+/*      .nav-home-title-text {
```
Comment (TrebuhD) [inline client/elements/sc-drawer-layout.html]: please don't commit commented-out code.

File: client/img/manifest.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+            "sizes": "192x192",
+            "type": "image/png"
+        },
+        {
+            "src": "/android-chrome-512x512.png",
+            "sizes": "512x512",
+            "type": "image/png"
+        }
+    ],
+    "theme_color": "#ffffff",
```
Comment (TrebuhD) [inline client/img/manifest.json]: The `theme_color` can be set to our primary accent color, that should make the android navbar have that color (this is just a random comment, we don't have to do that now).

File: client/index.html [hunk 3]
```diff
@@ -3,6 +3,11 @@
   <meta charset="UTF-8">
   <title>SuttaCentral</title>
   <link rel="icon" href="/img/favicon.ico">
+  <link rel="apple-touch-icon" sizes="180x180" href="/img/apple-touch-icon.png">
+  <link rel="icon" type="image/png" sizes="32x32" href="/img/favicon-32x32.png">
+  <link rel="icon" type="image/png" sizes="16x16" href="/img/favicon-16x16.png">
+  <link rel="manifest" href="/img/manifest.json">
+  <meta name="theme-color" content="#ffffff">
```
Comment (TrebuhD) [inline client/index.html]: shouldn't the theme color be our primary color?

File: client/index.html [hunk 4]
```diff
@@ -3,6 +3,11 @@
   <meta charset="UTF-8">
   <title>SuttaCentral</title>
   <link rel="icon" href="/img/favicon.ico">
+  <link rel="apple-touch-icon" sizes="180x180" href="/img/apple-touch-icon.png">
+  <link rel="icon" type="image/png" sizes="32x32" href="/img/favicon-32x32.png">
+  <link rel="icon" type="image/png" sizes="16x16" href="/img/favicon-16x16.png">
+  <link rel="manifest" href="/img/manifest.json">
```
Comment (TrebuhD) [inline client/index.html]: The manifest should not reside in the `img` directory, please move it up. When you do that, you'll have to update the paths in the manifest as well.

File: client/elements/sc-drawer-layout.html (hunk 1 above)
Comment (SamaneriVimala) [reply client/elements/sc-drawer-layout.html]: Oops ... accidentally forgot. Will change it.

**Pull request merged**